go/ct/spc/specification.go

```diff
@@ -1254,6 +1254,57 @@ func getAllRules() []Rule {
 		},
 	}...)
 
+	// --- BLOBHASH ---
+
+	rules = append(rules, rulesFor(instruction{
+		op:        BLOBHASH,
+		staticGas: 3,
+		pops:      1,
+		pushes:    1,
+		conditions: []Condition{
+			RevisionBounds(R13_Cancun, NewestSupportedRevision),
+			HasBlobHash(Param(0)),
+		},
+		parameters: []Parameter{NumericParameter{}},
+		effect: func(s *st.State) {
+			indexU256 := s.Stack.Pop()
+			if indexU256.Gt(NewU256(uint64(len(s.TransactionContext.BlobHashes)))) {
+				s.Stack.Push(NewU256(0))
```

Comment:
I am not sure if this is correctly modeled. If the condition is that a blobhash must exist, then the effect shall not provide the option to have an out of range case. 

I am not sure what should be doing instead, but according to the condition, this branch should never be executed.